_sass/_index-jumbotron.sass

```diff
@@ -1,3 +1,7 @@
 #index-jumbotron
     /* Background pattern from Subtle Patterns */
     background-image: url($img + "pw_pattern.png")
+
+.jumbotron
```

Comment:
Why is this applied to `.jumbotron` and not `#index-jumbotron`? Also `class="pt-0"` is a nice alternative

Reply:
no reason not to use `#index-jumbotron`, will include.
`.pt-0` - is this new in Bootstrap4? haha

Reply:
haha yes i think so, there's a ton of new utility classes